hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/state/migration/AccountStateTranslator.java

```diff
@@ -63,7 +66,10 @@ public static Account accountFromMerkle(
                         .getKeyAsBigInteger()
                         .toByteArray());
         return Account.newBuilder()
-                .accountNumber(account.getKey().longValue())
+                .accountId(AccountID.newBuilder()
+                        .accountNum(account.getKey().longValue())
+                        .realmNum(StaticProperties.getRealm())
```

Comment:
We discussed realm and shard values should come from HedraConfig at one of our Modularization calls.

Reply:
HederaConfig is not available here because we're not in a transaction handler. This class is for translation to/from the modularized codebase. Realm and Shard are always going to be zero because we do not expect to implement sharding until after we're completely free of non-modularized codebase.